server/controllers/articles.js

```diff
@@ -1 +1,129 @@
 import Article from "../models/Articles";
+export const getAllArticles = async (req, res, next) => {
+    try {
+        const articles = await Article.find();
+
+        return res.status(200).json({
+            success: true,
+            status: 200,
+            count: articles.length,
+            data: articles
+        });
+    } catch (err) {
+        return res.status(500).json({
+            success: false,
+            status: 500,
+            error: 'Server error'
+        });
+    }
+}
+
+export const createArticle = async (req, res, next) => {
+        const newArticle = new Article (
+            {
```

Comment:
How do you validate your inputs? If you forgot to validate, you can use the **JOI** package to validate.

Reply:
I was planning to create middlewares and validate my inputs using regular expressions.
but I'll try that JOI package.